lua/pixelui/core/cl_color.lua

```diff
@@ -16,27 +16,10 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 ]]
 
-local lerp = Lerp
-function PIXEL.LerpColor(t, from, to)
-    local newCol = Color(0, 0, 0)
-
-    newCol.r = lerp(t, from.r, to.r)
-    newCol.g = lerp(t, from.g, to.g)
-    newCol.b = lerp(t, from.b, to.b)
-    newCol.a = lerp(t, from.a, to.a)
-
-    return newCol
-end
-
-function PIXEL.CopyColor(col)
-    return Color(col.r, col.g, col.b, col.a)
-end
-
-function PIXEL.OffsetColor(col, offset)
-    return Color(col.r + offset, col.g + offset, col.b + offset)
-end
+PIXEL.HSLToColor = HSLToColor
+PIXEL.ColorToHSL = ColorToHSL
 
-function PIXEL.Hue2RGB(p, q, t)
+function PIXEL.HueToRGB(p, q, t)
```

Comment:
Document what "p,q,t" is, although it's relative to the orginal algorithm, it should be documented.

Reply:
The method was only ever used internally by PIXEL UI, so I've removed it now it has no use instead.